docs/book/src/clusterctl/overview.md

```diff
@@ -20,11 +20,15 @@ Below you can find a list of main clusterctl commands:
 * [`clusterctl move`](commands/move.md) Move Cluster API objects and all their dependencies between management clusters.
 * [`clusterctl alpha rollout`](commands/alpha-rollout.md) Manages the rollout of Cluster API resources. For example: MachineDeployments.
 
+### GitHub Personal Access Token
```

Comment:
I think this should mention why we want the PAT ~"Avoiding GitHub rate limiting"